### PR TITLE
Fix rubocop issues

### PR DESCRIPTION
**Pull request by ignaciojonas**

### CODE REVIEW

File: examples/ruby-api/Gemfile [hunk 1]
```diff
@@ -5,4 +5,4 @@ source 'http://rubygems.org'
 # gem "rails"
 gem 'sinatra', '~> 1.4'
 gem 'jwt', '~> 1.5'
-gem 'dotenv'
\ No newline at end of file
+gem 'dotenv'
```
Comment (falvarisqueta) [inline examples/ruby-api/Gemfile:8]: rubocop asks for a line break?

File: examples/ruby-api/config.ru [hunk 2]
```diff
@@ -1,2 +1,2 @@
 require './main'
-run Sinatra::Application
\ No newline at end of file
+run Sinatra::Application
```
Comment (falvarisqueta) [inline examples/ruby-api/config.ru]: Looks like this change is not necessary

File: examples/ruby-api/Gemfile (hunk 1 above)
Comment (ignaciojonas) [inline examples/ruby-api/Gemfile:8]: yes.

Comment (leosoubeste) [inline examples/ruby-api/Gemfile:8]: Looks good to me.

**Pull request merged**